microsite/data/plugins/dx.yaml

```diff
@@ -0,0 +1,10 @@
+---
+title: DX
+author: DX
+authorUrl: https::/getdx.com
```

Comment:
Think this might need an update :)

Reply:
Thank you! I'll fix haha 

And I'll go update our README's, we are no longer in a closed beta 👍🏻

Reply:
@benjdlambert I've removed those beta warnings from our package README 👍🏻